### ISSUE TITLE
Support for 'find my phone' functionality

### ISSUE THREAD

**Issue by arashpayan** [zood/george#31]
Implement it as a native feature because of memory limitations with libsodium running in (w)asm.js

**Comment by glocalglocal** [comment]
Is this feature still on the cards? It would be a great way to eliminate the need for Google's app, but this issue is open since 2018.

**Comment by RasheedAZ** [comment]
I hope this would work on Linux as well, since there is no dedicated app like this for Linux phones at the moment.

**Comment by arashpayan** [comment]
Yeah, it should work from any browser. The code to make the phone ring loudly (despite volume settings) when contacted by the owning user is already in the released Android app. This issue is just about building the web page where one can log in to actually initiate the ringing.
